content-src/components/DiscoveryStreamComponents/DSContextFooter/_DSContextFooter.scss

```diff
@@ -0,0 +1,100 @@
+$status-green: #058B00;
+
+.story-footer {
+  color: var(--newtab-text-secondary-color);
+  display: flex;
+  height: 24px;
+  inset-inline-start: 0;
+  margin-top: 12px;
+  align-items: center;
+  transition: transform 0.3s;
+  transform-style: preserve-3d;
+
+  .story-sponsored-label,
+  .status-message {
+    @include dark-theme-only {
+      color: $grey-40;
+    }
+
+    font-size: 13px;
+    line-height: 24px;
+    color: $grey-50;
+  }
+
+  .story-sponsored-label {
+    color: $grey-50;
+  }
+
+  .status-message {
+    display: flex;
+    align-items: center;
+    animation: rotateIn 0.3s ease-out;
+    height: 24px;
+
+    .story-badge-icon {
+      fill: $grey-50;
+      height: 16px;
+      margin-inline-end: 6px;
+      animation: color 2s ease-out;
+
+      &.icon-bookmark-removed {
+        background-image: url('../data/content/assets/icon-removed-bookmark.svg');
+      }
+    }
+
+    .story-context-label {
+      color: $grey-50;
+      flex-grow: 1;
+      font-size: 13px;
+      line-height: 24px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      animation: color 2s ease-out;
```

Comment:
animation property for  .story-badge-icon and .story-context-label  can be moved below similar to dark-theme handling